### PR TITLE
Redirect v0.9-0.12 docs to the relevant 1.0 doc

### PR DESCRIPTION
**Pull request by rkuchan**
The warning at the top of version v0.x though 0.13 now links to the relevant 1.0 doc (for Telegraf, InfluxDB, Chronograf, and Kapacitor).

In addition this:

* Adds the `max-series-per-database` setting to the configuration doc, #710.
* Goes through and fixes any broken links in the InfluxDB v1.0 docs.
* Fixes the NSQ output link the Telegraf 1.0 docs, #729.

Notes:

* InfluxDB versions 0.6-0.8 still link to the 1.0 landing page
* The commit history on this is messed up 😓

### CODE REVIEW

File: content/influxdb/v1.0/administration/config.md [hunk 1]
```diff
@@ -369,6 +369,25 @@ The compact full write cold duration is the duration at which the engine will co
 The maximum points per block is the maximum number of points in an encoded block in a TSM file.
 Larger numbers may yield better compression but could incur a performance penalty when querying.
 
+### max-series-per-database = 1000000
+
+The maximum number of [series](/influxdb/v1.0/concepts/glossary/#series) allowed
+per database.
+The default is setting is `1M`.
```
Comment (beckettsean) [inline content/influxdb/v1.0/administration/config.md]: I would do either

The default setting is `1000000`
or
The default setting is one million.

File: content/influxdb/v1.0/external_resources/index.md [hunk 2]
```diff
@@ -4,3 +4,17 @@ title: External Resources
 
 But wait, there's more!
 Check out these resources to learn more about InfluxDB.
+
```
Comment (beckettsean) [inline content/influxdb/v1.0/external_resources/index.md]: Also add https://www.influxdata.com/virtual-training-courses/ and https://www.influxdata.com/videos-training/?

Probably https://www.influxdata.com/events/ too

File: content/influxdb/v1.0/introduction/installation.md [hunk 3]
```diff
@@ -271,8 +271,3 @@ When using non-standard directories for InfluxDB data and configurations, also b
 chown influxdb:influxdb /mnt/influx
 chown influxdb:influxdb /mnt/db
 ```
-
-## Nightly and Development Versions
```
Comment (beckettsean) [inline content/influxdb/v1.0/introduction/installation.md]: Did we take the nightlies out of the package repos?

File: content/influxdb/v1.0/troubleshooting/query_management.md [hunk 4]
```diff
@@ -16,13 +16,7 @@ This document covers how to:
 
 * [List currently-running queries with `SHOW QUERIES`](/influxdb/v1.0/troubleshooting/query_management/#list-currently-running-queries-with-show-queries)
 * [Stop currently-running queries with `KILL QUERY`](/influxdb/v1.0/troubleshooting/query_management/#stop-currently-running-queries-with-kill-query)
-* [Configure query management](/influxdb/v1.0/troubleshooting/query_management/#configuration-settings-for-query-management)
```
Comment (beckettsean) [inline content/influxdb/v1.0/troubleshooting/query_management.md]: 👍

File: content/kapacitor/v0.2/tick/reduce_node.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: ReduceNode
 note: Auto generated by tickdoc
-
+newversionredirect: /nodes/reduce_node/
```
Comment (beckettsean) [inline content/kapacitor/v0.2/tick/reduce_node.md]: There is no reduce node any longer. Should probably redirect to `/nodes/` like map_node does.

File: content/influxdb/v1.0/introduction/installation.md (hunk 3 above)
Comment (rkuchan) [inline content/influxdb/v1.0/introduction/installation.md]: @beckettsean, I don't see them on the downloads page anymore so I just removed this section. I can add nightlies to the new section that'll come with  https://github.com/influxdata/docs.influxdata.com/issues/737.

**Pull request merged**